examples/keccak/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "keccak"
```

Cargo gets unhappy when there are multiple crates with the same name. `keccak` is crate on `crates.io`, so having this something unique would be a good.
```suggestion
name = "risc0-keccak-example"
```